build.gradle.kts

```diff
@@ -136,6 +136,7 @@ configure<IdeaModel> {
     module.apply {
         generatedSourceDirs.add(file("gen"))
         excludeDirs.add(file(the<IntelliJPluginExtension>().sandboxDirectory))
+        isDownloadSources = true
```

Comment:
I don't think this will make a difference when importing the project using IntelliJ IDEA, does it?